var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -211,11 +225,12 @@ def cmake_args(self):
                 from_variant("TPL_ENABLE_Pthread", "thread_safe"),
                 from_variant("TPL_ENABLE_X11", "x11"),
                 from_variant(project_name_base + "_ENABLE_Fortran", "fortran"),
+                define(project_name_base + "_ENABLE_SEACAS", True),
```

Comment:
(question) this seems like it should be correct everywhere, but because this affects builds outside of Windows, what does it do?

Reply:
I'm honestly not sure what it does on Linux (IIRC this is true by default on linux so it's a noop), I just copied the build arguments used in the `seacas` CI as that's the only configuration that is guaranteed to pass on Windows.

Reply:
> IIRC this is true by default on linux so it's a noop

Can you confirm that? This is the main thing I want to be sure about before merging.

Reply:
Looks like it's `ON` everywhere by default, every other reference in the CMake is a query for that variable state.  So it's also a no-op on Windows, I'll just take it out.

https://github.com/sandialabs/seacas/blob/master/packages/seacas/CMakeLists.txt#L71

Reply:
The SEACAS package is "snapshotted" into some other projects (e.g. Trilinos), and we use that symbol inside Trilinos to turn on/off SEACAS. Just to minimize changes between the "standalone" seacas build and the Trilinos build and some other packages that embed SEACAS, we have that variable here...

Reply:
IIRC, there is some setting in TriBITs which is the "cmake wrapper" used in SEACAS and in Trilinos which defaults to not building on windows, so we explicitly need to set the `project_name_base + "_ENABLE_SEACAS"` variable to `True` on windows and it defaults to `True` on linux and macos.

Reply:
Thanks for that explanation! It sounds like this is fine then.

Reply:
@scheibelp I think it's probably ok to leave in this case?

Reply:
Nevermind